scripts/deploy-pr.sh

```diff
@@ -71,8 +71,9 @@ fi
 gh_auth
 
 # get branch name and pr number
-branchName=$(gh pr view --json headRefName | jq -r .headRefName)
-prNumber=$(gh pr view --json number | jq -r .number)
+info=$(gh pr status --repo=coder/coder --json headRefName,number --jq '.createdBy[0]')
```

Comment:
```suggestion
info=$(gh pr status --json headRefName,number --jq '.currentBranch')
```

Reply:
This seems to work with minimum changes.

Reply:
Doesn't work for me.

Reply:
You are running it from your fork. And I am running it from coder/coder with only your fork's branch checked out. `gh` is indeed mysterious.